trt_yolo_ros/src/trt_yolo_ros/trt_yolo_ros.py

```diff
@@ -72,23 +75,20 @@ def _init_topics(self):
         self._pub_viz = rospy.Publisher(
             topic, Image, queue_size=queue_size, latch=latch
         )
-        # Subscriber
-        topic, queue_size = self._read_subscriber_param("image")
-        self._image_sub = rospy.Subscriber(
-            topic, Image, self._image_callback, queue_size=queue_size, buff_size=2 ** 24
+        # Image Subscriber
+        for i in range(self.num_cameras):
+            topic, queue_size = self._read_subscriber_param("image" + str(i)) 
+            self._image_sub = rospy.Subscriber(
+                topic, Image, self._image_callback, queue_size=queue_size, buff_size=2 ** 24
         )
         rospy.logdebug("[trt_yolo_ros] publishers and subsribers initialized")
 
     def _image_callback(self, msg):
         """ Main callback which is saving the last received image """
-        self.msg = None
-        try:
-            self.image = self._bridge.imgmsg_to_cv2(msg, "bgr8")
-            self.msg = msg
-            rospy.logdebug("[trt_yolo_ros] image received")
-        except CvBridgeError as e:
-            print(e)
-
+        if msg.header != None:
```

Comment:
Can you replace `==` with `is` more info under this [link](https://stackoverflow.com/questions/2020598/in-python-how-should-i-test-if-a-variable-is-none-true-or-false/2021257#2021257)